aldryn_client/localdev/main.py

```diff
@@ -82,10 +83,18 @@ def create_workspace(client, website_slug, path=None):
         check_call(docker_compose('rm', '-f', 'db'))
 
     click.secho('creating new database container', fg='green')
-    load_database_dump(client, path, recreate=True)
+    load_database_dump(client, path)
 
     click.secho('sync and migrate database', fg='green')
-    check_call(docker_compose('run', 'web', './migrate.sh'))
+
+    if is_windows():
+        # interactive mode is not yet supported with docker-compose
+        # on windows. that's why we have to call it as daemon
+        # and just wait a sane time
+        check_call(docker_compose('run', '-d', 'web', 'start', 'migrate'))
+        sleep(30)
```

Comment:
any chance to run ``docker logs`` on the migrate command to get the output anyway? If not, lets at least show a message that output is suppressed on windows.